.changeset/stale-hairs-wonder.md

```diff
@@ -0,0 +1,5 @@
+---
+'@hono/zod-openapi': minor
```

Comment:
This should be `patch` instead of `minor` because it is a bug fix.

Reply:
oops, will fix

Reply:
done. the cli didn't include the `patch` option for me, only minor